src/github/api.rs

```diff
@@ -303,7 +303,6 @@ impl GitHub {
                         memberships.insert(
                             edge.node.database_id,
                             TeamMember {
-                                id: edge.node.database_id,
```

Comment:
Nit: we should remove `databaseId` from the GraphQL query above as well.

Reply:
Don't we still need that for the key?

Reply:
Oh, if we still need it let's keep it! I thought it wasn't used anywhere else :sweat_smile: